src/scrape_up/github/respository.py

```diff
@@ -267,3 +268,26 @@ def get_contributors(self):
         except:
             message = "Oops! No contributors found"
             return message
+
+    
+    def get_special_repo(self):
+        """
```

Comment:
Just a minor change is needed, rename the function to `get_readme()`.

Reply:
Okay DONE. 😄